san/sanbase_graphql.py

```diff
@@ -1,4 +1,9 @@
 import iso8601
+import datetime
+
+DEFAULT_TO_DATE = datetime.datetime.now().isoformat()
+DEFAULT_FROM_DATE = (datetime.datetime.now() - datetime.timedelta(days=365)).isoformat()
```

Comment:
I think this value will be calculated when the library loads and won't change from there, so if the scripts runs for more than 1 day the default will point to yesterday. May be we can have these defaults as lambdas and calculate them when they are used?